apps/vessel-history/features/datasets/datasets.slice.ts

```diff
@@ -136,16 +140,20 @@ export const selectAll = createSelector([baseSelectAll], (datasets) => {
   const vesselInfo: Dataset[] = datasets
     .filter((d) => d.category === 'vessel' && d.subcategory === 'info')
     // Inject Proto Gaps Dataset
-    .map((d) => ({
-      ...d,
-      relatedDatasets: [
-        ...d.relatedDatasets,
-        {
-          id: 'proto-global-gaps-events:v20201001',
-          type: DatasetTypes.Events,
-        },
-      ],
-    }))
+    .map((d) =>
+      IS_STANDALONE_APP
+        ? d
+        : {
+            ...d,
+            relatedDatasets: [
+              ...d.relatedDatasets,
+              {
+                id: 'proto-global-gaps-events:v20201001',
+                type: DatasetTypes.Events,
+              },
+            ],
+          }
+    )
```

Comment:
It would be more convenient to verify this only once and not on every dataset mapping since the logic will not change for different datasets. I'd extract this callback to a function and use it here, eg: 
```typescript
  const mapDataset =  IS_STANDALONE_APP
        ? (d) => d
        : (d) => {
            ...d,
            relatedDatasets: [
              ...d.relatedDatasets,
              {
                id: 'proto-global-gaps-events:v20201001',
                type: DatasetTypes.Events,
              },
            ],
          }
  const vesselInfo: Dataset[] = datasets
    .filter((d) => d.category === 'vessel' && d.subcategory === 'info')
    // Inject Proto Gaps Dataset
    .map(mapDataset)
```

Reply:
this code is not needed, it was a path to support gaps, now is included in the dataview